src/integTest/kotlin/UIComponentFinder.kt

```diff
@@ -0,0 +1,24 @@
+import java.awt.Component
+import java.awt.Container
+import kotlin.reflect.KClass
+import kotlin.reflect.cast
+
+object UIComponentFinder {
+
+    @OptIn(ExperimentalStdlibApi::class)
```

Comment:
Interesting, what's that?

Reply:
It's easy to find, just comment it and Idea will show you ;)
Actually it's needed for KClass<T>.cast() call as it's for some reason still an experimental function